redirects-www.conf

```diff
@@ -250,6 +250,10 @@ rewrite ^/rad/(.*) https://transition.fec.gov/rad/$1 redirect;
 rewrite ^/pubrec/(.*) https://transition.fec.gov/pubrec/$1 redirect;
 rewrite ^/agenda/([0-9]+)/documents/(.*) /resources/updates/agendas/$1/$2 redirect;
 rewrite ^/agenda/([0-9]+)/(.*).pdf /resources/updates/agendas/$1/$2.pdf redirect;
+rewrite ^/agenda/agendas2003/(.*).pdf https://www.fec.gov/resources/updates/agendas/2003/$1 redirect;
+rewrite ^/agenda/agendas2002/(.*).pdf https://www.fec.gov/resources/updates/agendas/2002/$1 redirect;
+rewrite ^/agenda/agendas2001/(.*).pdf https://www.fec.gov/resources/updates/agendas/2001/$1 redirect;
+rewrite ^/agenda/agendas2000/(.*).pdf https://www.fec.gov/resources/updates/agendas/2000/$1 redirect;
```

Comment:
Almost there, just need to add the `.pdf` extension on the end of the resulting redirect. This will need to be done to all the resulting URLs. So like this https://www.fec.gov/resources/updates/agendas/2003/$1.pdf

Reply:
Ahhh! And you told me this before. Thanks for being so patient with me! It's added to all of them.